src/transient-buffer-heap.cpp

```
@@ -0,0 +1,349 @@
+}
+
+Result TransientBufferHeap::allocate(Size size, Chunk* outChunk)
+{
+    if (!outChunk)
+        return SLANG_E_INVALID_ARG;
+    *outChunk = {};
+    if (size == 0)
+        return SLANG_E_INVALID_ARG;
+
+    const Size alignedSize = alignAllocationSize(size);
+    if (alignedSize > m_desc.maxPageSize)
```

Cppcheck (2.21.0)

[style] The function 'getAllocation' is never used.

(unusedFunction)